backends/platform/iphone/iphone_video.mm

```diff
@@ -158,31 +166,74 @@ - (void)createContext {
 	}
 }
 
+- (void)setupGestureRecognizers {
+	UISwipeGestureRecognizer *swipeRight = [[UISwipeGestureRecognizer alloc] initWithTarget:self action:@selector(twoFingersSwipeRight:)];
```

Comment:
Looks like these swipe gesture recognizers have not been available in older iOS versions (we use SDK version 3.1.2 to support all generations of devices):
```
backends/platform/iphone/iphone_video.mm:170:2: error: unknown type name 'UISwipeGestureRecognizer'
        UISwipeGestureRecognizer *swipeRight = [[UISwipeGestureRecognizer alloc] initWithTarget:self action:@selector(twoFingersSwipeRight:)];     
        ^
```